source/cl/source/extension/include/extension/intel_unified_shared_memory.h

```diff
@@ -192,6 +197,13 @@ class host_allocation_info final : public allocation_info {
   /// @return A matching Mux buffer object on success, or nullptr on failure
   mux_buffer_t getMuxBufferForDevice(cl_device_id device) const override;
 
+  /// @brief Return the USM memory type for this allocation
+  ///
+  /// @return The enum variant corresponding to this memory allocation type
+  virtual cl_unified_shared_memory_type_intel getMemoryType() const override {
```

Comment:
The `virtual` keyword is redundant here. We are not consistent in whether we leave it in or take it out; the function right above this one has `override` without `virtual`, but there are plenty of functions that use both as well. Do we have a documented style that we should adhere to for new code?

Reply:
I think you are right. I think we tend to rely on clang-tidy etc to bring up things.